alpha/lib/data/live/kLiveEntryRecordingOptions.php

```diff
@@ -23,7 +23,12 @@ class kLiveEntryRecordingOptions
 	 * @var boolean
 	 */
 	protected $shouldCopyThumbnail;
-	
+
+	/**
+	 * @var boolean
```

Comment:
Please use KalturaNullableBoolean instead of boolean and in your code make sure that in all places null will be handled as identical to false.